packages/c/clientworker.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "clientworker",
+  "filename": "dist/cw.js",
+  "description": "A simple, globally hijacked, easily customizable, Service Worker-based front-end worker",
+  "homepage": "https://clientworker.js.org",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChenYFan/ClientWorker.git"
+  },
+  "keywords": [
+    "ServiceWorker",
+    "ClientWorker",
+    "Accelerate"
+  ],
+  "license": "GPL-3.0-or-later",
+  "autoupdate": {
+    "source": "npm",
+    "target": "clientworker",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "@(dist|main|doc/docs/.vitepress/dist)/*.js"
```

Comment:
Why are we including the JavaScript for your VitePress site? This doesn't seem at all relevant to what end users would want to be loading on their own sites?

Reply:
> Why are we including the JavaScript for your VitePress site? This doesn't seem at all relevant to what end users would want to be loading on their own sites?

Oh, this is a small mistake. When I copied from glob, I planned to submit the clientworker as an example, but I forgot that there are a lot of static resources underneath.